packages/site-kit/src/lib/docs/DocsCopyCodeButton.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+	import Icon from '$lib/components/Icon.svelte';
+	import { cubicOut } from 'svelte/easing';
+	import { fade } from 'svelte/transition';
+
+	/** @type {string} */
+	export let code;
+
+	let copying = false;
+
+	function copy() {
+		try {
+			navigator.clipboard.writeText(code);
```

Comment:
@PuruVJ this returns `Promise`, needs async catch I believe.

ref: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText#return_value